recipe/run_conda_forge_build_setup_linux

```diff
@@ -25,6 +25,27 @@ if [[ "$CONFIG" == *_pypy* ]]; then
   conda config --set channel_priority strict
 fi
 
+# the upstream image nvidia/cuda:9.2-devel-centos6 (on which linux-anvil-cuda:9.2 is based)
+# does not contain libcuda.so; it should be installed in ${CUDA_HOME}/compat-${CUDA_VER},
+# however cuda-compat-${CUDA_VER}.x86_64.rpm is only packaged for 10.x; we abuse
+# cuda-compat-10.0 for this, since the actual dependency containing libcuda for 9.2
+# (xorg-x11-drv-nvidia-libs) pulls in a huge amount of dependencies;
+# this cannot be fixed in the conda-forge linux-anvil-cuda images for licensing reasons
+# (cannot add cuda-package in our image layers), so we add it here for CI purposes only.
+if [[ ! -z "$CUDA_HOME" && -d /usr/local/cuda-9.2 ]]; then
+  # see https://developer.download.nvidia.com/compute/cuda/repos/rhel6/x86_64/
+  curl -O https://developer.download.nvidia.com/compute/cuda/repos/rhel6/x86_64/cuda-repo-rhel6-9.2.148-1.x86_64.rpm && \
+  sudo yum localinstall -y cuda-repo-*.rpm && \
+  rm cuda-repo-*.rpm && \
+  sudo yum install -y cuda-compat-10-0.x86_64 && \
+  # note that $CUDA_HOME is just a symlink to /usr/local/cuda-${CUDA_VER}, and yum installs in that folder
+  sudo mv /usr/local/cuda-10.0/* /usr/local/cuda-9.2 && \
```

Comment:
Could we try the following approach:
- keep /usr/local/cuda-10.0/compat as it is
- in linux-anvil-cuda/Dockerfile append /usr/local/cuda-10.0/compat to /etc/ld.so.conf.d/cuda-$CUDA_VER.conf only when CUDA_VER=9.2
?

Reply:
@isuruf 
What do you think about this idea? I dislike not being able to fix this correctly here, but without proper elevation - maybe we could add these two specific commands to the sudoer-list in the image?

Reply:
The sudoers-route works (I tested it), and should lead to IMO the cleanest solution all-around: https://github.com/conda-forge/docker-images/pull/138.

Reply:
I don't think this is safe to do. If a user builds a CUDA 9.2 package with this CUDA 10.0 compat `libcuda.so` this can potentially break things on a driver 396 install for example as this compat I believe is based on driver 410. I could be mistaken about this though.

@teju85 @harrism do you know if this is the case?

Reply:
The user is not building against `CUDA 10.0`. It's just loading the application using cuda compat 10.0. This is useful for checking that packages can be loaded on CI without GPUs.

Reply:
Sorry maybe I'm unclear of the goal of moving this into the `/usr/local/cuda-9.2` folder. Is this added to `LD_LIBRARY_PATH` or something by default where it would get picked up or is this being added for someone to explicitly point to?

Reply:
Yes, it's added to ld.conf which has the almost the same effect as `LD_LIBRARY_PATH`

@h-vetinari, how about not moving that and adding `/usr/local/cuda-10.2/compat` to ld.conf in cuda 9.2 image?

Reply:
> Yes, it's added to ld.conf which has the almost the same effect as `LD_LIBRARY_PATH`
> 
> @h-vetinari, how about not moving that and adding `/usr/local/cuda-10.2/compat` to ld.conf in cuda 9.2 image?

Won't this basically cause symbols from a newer driver library to be used during building packages which aren't backwards compatible? I believe CUDA 10.0 compat is based on driver 410, CUDA 10.2 compat is based on driver 440, but CUDA 9.2 can use driver 396 which presents possible issues from lack of backwards compatibility.

Reply:
Nope. We are using the stub library from 9.2 to build and the 10.0 compat is used only for loading

Reply:
> @h-vetinari, how about not moving that and adding /usr/local/cuda-10.0/compat to ld.conf in cuda 9.2 image?

This is what @pearu suggested at the top of this review thread. Fine by me.

Reply:
See here: https://github.com/conda-forge/docker-images/pull/139